lib/Support/LoweringOptions.cpp

```diff
@@ -113,6 +113,8 @@ void LoweringOptions::parse(StringRef text, ErrorHandlerT errorHandler) {
       emitWireInPorts = true;
     } else if (option == "emitBindComments") {
       emitBindComments = true;
+    } else if (option == "doNotEmitVersionComment") {
+      doNotEmitVersionComment = true;
```

Comment:
I think this should have an entry in `toString` as well (like the other options)?

Reply:
done